lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeedFormatter.cs

```diff
@@ -48,20 +43,20 @@ public SyndicationFeed Feed
 
         public abstract bool CanRead(XmlReader reader);
 
-        public abstract void ReadFrom(XmlReader reader);
+        public abstract Task ReadFromAsync(XmlReader reader);
 
         public override string ToString()
         {
             return String.Format(CultureInfo.CurrentCulture, "{0}, SyndicationVersion={1}", this.GetType(), this.Version);
         }
 
-        public abstract void WriteTo(XmlWriter writer);
+        public abstract Task WriteTo(XmlWriter writer);
 
         internal static protected SyndicationCategory CreateCategory(SyndicationFeed feed)
         {
             if (feed == null)
             {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("feed");
+                throw new ArgumentNullException("feed");
             }
```

Comment:
I'm going to stop adding this comment repeatedly, please fix all parameter name text constants with nameof(parameterName)